### PR TITLE
change view call to get-events method

### PR DESCRIPTION
**Pull request by aramis-eb**

### CODE REVIEW

File: search_events_app/views.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 from django.shortcuts import render
 from django.views.generic.list import ListView
 
-from .services.dummy_api import DummyApi
+from .services.api_service import get_events
```
Comment (julianrofrano-eb) [inline search_events_app/views.py:4]: Please use absolute import here

File: search_events_app/models/event.py [hunk 2]
```diff
@@ -8,3 +8,5 @@ def __init__(self, name, url, country=None, feature=None, **kwargs):
             self.feature = []
         else:
             self.feature = feature
+        for key, value in kwargs.items():
```
Comment (julianrofrano-eb) [inline search_events_app/models/event.py:11]: Do you think this would need a test?

File: search_events_app/views.py [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 
 # Create your views here.
 class EventListView(ListView):
     template_name = 'event_list.html'
 
     def get_queryset(self):
-        return DummyApi().get()
+        return get_events
```
Comment (arielaranda-eb) [inline search_events_app/views.py:12]: isn't `get_events` a method?
is that what you want to return?

File: search_events_app/models/event.py (hunk 2 above)
Comment (aramis-eb) [reply search_events_app/models/event.py:11]: Done in the new commit

File: search_events_app/views.py (hunk 3 above)
Comment (aramis-eb) [reply search_events_app/views.py]: Yes, Done in the new commit!

**Pull request merged**